bindinfo/bind_test.go

```diff
@@ -719,6 +719,37 @@ func (s *testSuite) TestDefaultDB(c *C) {
 	tk.MustQuery("show session bindings").Check(testkit.Rows())
 }
 
+func (s *testSuite) TestEvolveInvalidBindings(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	s.cleanBindingEnv(tk)
+	tk.MustExec("use test")
+	tk.MustExec("drop table if exists t")
+	tk.MustExec("create table t(a int, b int, index idx_a(a))")
+	tk.MustExec("create global binding for select * from t where a > 10 using select /*+ USE_INDEX(t) */ * from t where a > 10")
+	// Manufacture a rejected binding by hacking mysql.bind_info.
+	tk.MustExec("insert into mysql.bind_info values('select * from t where a > ?', 'select /*+ USE_INDEX(t,idx_a) */ * from t where a > 10', 'test', 'rejected', '2000-01-01 09:00:00', '2000-01-01 09:00:00', '', '')")
```

Comment:
Change the `rejected` to `pending verify`?

Reply:
It doesn't matter for the case?

Reply:
If the status is `rejected`. It doesn't evolve this task. So the `admin evolve baseline` is meaningless. As a result, the binding will be never checked during evolution.

Reply:
In `getOnePendingVerifyJob`, binding in both `pending verify` and `rejected` status would be chosen for evolving. The `update_time` of the inserted binding is `2000-01-01 09:00:00` so it will be chosen.

Reply:
Sry, I didn't see the update time.